lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -998,38 +1017,70 @@ namespace Js
     bool WebAssemblyArrayBuffer::IsValidVirtualBufferLength(uint length) const
     {
 #if ENABLE_FAST_ARRAYBUFFER
-        return true;
+        return CONFIG_FLAG(WasmFastArray);
 #else
         return false;
 #endif
     }
 
-    ArrayBuffer * WebAssemblyArrayBuffer::TransferInternal(uint32 newBufferLength)
+    WebAssemblyArrayBuffer* WebAssemblyArrayBuffer::GrowMemory(uint32 newBufferLength)
```

Comment:
I wanted to do this initially, but there are multiple places in there that can throw, so either I catch the exception in GrowMemory or in WebAssemblyMemory::GrowInternal